tests/post_training/README.md

```diff
@@ -0,0 +1,32 @@
+# Post-training Quantization Confirmance Suite
+This is the test suite that takes PyTorch Timm models and runs post-training quantization on ImageNet dataset for the following three representations:
+- PyTorch
+- ONNX
+- OpenVINO
+
+The outcome of each quantization step is accuracy and performance with OpenVINO. The source representation is converted to OpenVINO IR at this step.
+
+## Installation
+```
+pip install -r requirements.txt
+python setup.py install
+```
+
+## Data preparation
+Since Torchvision `ImageFolder` class is used to work with data the ImageNet validation dataset should be structured accordingly. Below is an example of the `val` folder:
+```
+n01440764
+n01695060
+n01843383 
+...
+```
+
+## Usage
+Once the evnironment is installed use the following command to run the test:
+```
+NUM_VAL_THREADS=8 pytest --data=<path_to_imagenet_val_folder> --output=./tmp tests/post_training/test_quantize_conformance.py
+```
+
+`NUM_VAL_THREADS` environment variable controls the number of parallel streams when validating the model.
```

Comment:
This was done intentionally since it is an environment attribute not the test.

Reply:
It is a parameter influencing the test that you seem to want to expose to the user of the test. This should be done via the conventional way of defining a pytest option, since we control the test ourselves. Relying on environment variables is essentially using global variables. You don't use an environment variable for determining the output directory, do you?